content/en/docs/concepts/containers/images.md

```diff
@@ -44,6 +44,16 @@ There are additional rules about where you can place the separator
 characters (`_`, `-`, and `.`) inside an image tag.  
 If you don't specify a tag, Kubernetes assumes you mean the tag `latest`.
 
+## Image pull per runtime class
```

Comment:
I would move this to just before the heading `## Serial and parallel image pulls`, and make it a 3rd level (`###` heading).